pkg/reconciler/openshift/tektonhub/extension.go

```diff
@@ -404,6 +404,20 @@ func (oe openshiftExtension) SetAuthBaseURL(ctx context.Context, th *v1alpha1.Te
 			return err
 		}
 		th.Status.SetAuthRoute(fmt.Sprintf("https://%s", authRoute))
+
+		if string(secret.Data["AUTH_BASE_URL"]) != th.Status.AuthRouteUrl {
```

Comment:
we want to put it in data or stringData?

Reply:
So with the current patch we have put in stringData but while updating/creating the secret, it gets encoded and added in the data

Reply:
so should we put directly in data ?
i.e. instead of `secret.StringData["AUTH_BASE_URL"] = th.Status.AuthRouteUrl` it will be `secret.Data["AUTH_BASE_URL"] = []byte(th.Status.AuthRouteUrl)`